release/models/gnsi/openconfig-gnsi-credentialz.yang

```diff
@@ -256,17 +270,18 @@ module openconfig-gnsi-credentialz {
     }
   }
 
-  // Augments section.
-
-  augment "/oc-sys:system" {
-    when "oc-sys-grpc:grpc-servers/oc-sys-grpc:grpc-server" +
-        "/oc-sys-grpc:config[contains(oc-sys-grpc:services, 'oc-gnsi:GNSI')]" +
-        "/oc-sys-grpc:enable = 'true'";
+  grouping credentialz-base {
     description
-      "Console credentials freshness data.";
+      "";
+
+    container credentialz {
+      description
+        "";
 
-    uses console-config-state;
+      uses console-config-state;
+    }
   }
+
```

Comment:
the augments below should be refactor to follow the style you recommended.  Perhaps you didn't do this yet because you were waiting for comments on the recommended style.

Reply:
Agreed - also there are a number of issues within the gNSI modeling that have been discovered recently that will have to aggregate into a separate discussion/PR